scripts/pages/map-selector/map-selector.ts

```diff
@@ -61,7 +61,9 @@ class MapSelectorHandler implements OnPanelLoad {
 			unranked: $<Button>('#MapListUnranked'),
 			beta: $<Button>('#MapListBeta')
 		},
-		refreshIcon: $<Image>('#RefreshIcon')
```

Comment:
Don't see any reason to do this in JS, can use `visible="false` or the `hidden` class in XML.